src/components/drawing/index.js

```diff
@@ -678,3 +678,13 @@ drawing.setPointGroupScale = function(selection, x, y) {
 
     return scale;
 };
+
+drawing.measureText = function(tester, text, font) {
+    var dummyText = tester.append('text')
+        .text(text)
+        .call(drawing.font, font);
+
+    var bbox = drawing.bBox(dummyText.node());
+    dummyText.remove();
+    return bbox;
+};
```

Comment:
I see why you did it this way, but it seems a little redundant, we're making a new node only to clone it in the global tester, then we delete them both. If this ends up getting called a lot, for performance we may want to break up `drawing.bBox` in such a way that this element can get created in the global tester in the first place.